scripts/create_ocd_ids.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import print_function
+import csv
+"""
+Program to generate OCD IDs for the 2019 Indian Vidhan Sabha elections.
+"""
+
+contests = {"hr": "Haryana", "mh": "Maharashtra"}
+columns = ["id", "name"]
+country = "in"
+election = "Vidhan Sabha"
+write_table = []
+new_file = True
+
+# replace for other countries
+district_replacements = {
+    "district": " ",
+    "Yamuna Nagar": "Yamunanagar",
+    "Gondia": "Gondiya",
+    "Gurugram": "Gurgaon",
+    "Mewat": "Nuh",
+    "Dhamangaon": "Dhamamgaon",
+    "Deolali": "Devlali",
+    "Chikhali": "Chikhli",
+    "Ballabgarh": "Ballabhgarh",
+    "Kalanwali": "Kalawali",
+    "Navapur": "Nawapur",
+    "Sangole": "Sangola",
+    "Yewla": "Yevla",
+    "Jat": "Jath",
+    "Kalvan": "Kalwan",
+    "Kasba Peth": "Kasbapeth"
+}
+
+const_replacements = {
```

Comment:
Why are these being replaced?

Reply:
With the electoral districts in India have changed frequently in the last decade, so there's a lot of erroneous information out there. Some of it has made its way to wikipedia, which is unfortunately the only place I'd found abbreviations. To reconcile the differences between the the ultimate source of truth (https://affidavit.eci.gov.in) and the spreadsheet of abbreviations to districts, I use this dictionary. (Actually, this particular set of replacements is going to be taken out of this PR as a more concrete source of constituencies has been found with all corresponding states and districts: https://electoralsearch.in